errors/daemon.go

```diff
@@ -226,12 +226,21 @@ var (
 	})
 
 	// ErrorCodeIPCRunning is generated when we try to join a container's
-	// IPC but its not running.
+	// IPC but it's not running.
 	ErrorCodeIPCRunning = errcode.Register(errGroup, errcode.ErrorDescriptor{
 		Value:          "IPCRUNNING",
 		Message:        "cannot join IPC of a non running container: %s",
 		Description:    "An attempt was made to join the IPC of a container, but the container is not running",
-		HTTPStatusCode: http.StatusInternalServerError,
+		HTTPStatusCode: http.StatusConflict,
+	})
+
+	// ErrorCodeIPCRestarting is generated when we try to join a container's
+	// IPC but it's restarting.
+	ErrorCodeIPCRestarting = errcode.Register(errGroup, errcode.ErrorDescriptor{
```

Comment:
Yeah, I see that. However I wonder if that's enough, because this is for two flags and I think maybe it's better to give a more specific error description to avoid confusion :)